Assets/Engine/Inventory/Item.cs

```diff
@@ -85,6 +85,12 @@ public Container Container
         public void Awake()
         {
             sprite = null;
+
+            // Add a warning if an item is not on the Item layer (layer 16).
+            if (gameObject.layer != 16)
+            {
+                Debug.LogWarning("Item " + Name + " is on layer " + gameObject.layer);
```

Comment:
Inconsistent spacing on `"Item "`?

Reply:
This is simply the format of the warning message sentence. Removing the space would result in a message like: "ItemMug is on layer 0".

Reply:
Oh, ok.